docs/azure/sdk/snippets/dependency-injection/WebApplicationBuilder/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿#region snippet_WebApplicationBuilder
+using Azure.Identity;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Azure;
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+#region snippet_WebApplicationBuilder
+builder.Services.AddAzureClients(async clientBuilder =>
```

Comment:
The same discussion from HostBuilder applies here.   `AddAzureClients` accepts an `Action<T>` so there's no way to make this async safely.